### DISCUSSION TITLE
Thanks for the graph db resources

### DISCUSSION THREAD

**Discussion by dreddnafious** [digoal/blog#119] [General]
The content I am finding from you is from approx 2019. I am curious if you continued the recursive search strategy and continued to explore graph db in postgres using it. I am looking for any additional resources or insights you could point me to. If you had a book on the subject I would buy it.

I am developing a stealth social graph similar to linkedin and I am hoping to use postgres to drive the data.

Once again, thank you for your posts and I would really appreciate any more resources you have found helpful on the subject.

**Comment by digoal** [comment]
Sorry, I just saw your message. Has your project been launched? What is the data scale and does PostgreSQL's recursion meet the requirements?

**Comment by digoal** [comment]
here is some info about my testing:
https://github.com/digoal/blog/blob/master/201801/20180102_04.md
